modules/ROOT/pages/spring-data-neo4j.adoc

```diff
@@ -1,318 +1,478 @@
+Note the command ready to run in the prompt (`:play movies`).
+Execute that command, and an interactive slidedeck will appear just below the command line.
+On the second slide of that guide, execute the long Cypher statement to fill your database with our movie test data.
+
+[#create-project]
+== Create a new Spring Boot project
+
+The easiest way to set up a Spring Boot project is with the Spring Initializr at https://start.spring.io[start.spring.io^].
+It is also integrated in the major IDEs, in case you prefer not to use the website.
+
+Then, you can change the default group, artifact, name, and description for the project.
+Next, we can choose our project dependencies.
+We can search for and add the `Neo4j` and `Spring Reactive Web` starter to get what we need to create a reactive, Spring-based web application.
+
+Once those steps are complete, we can click the `Generate` button at the bottom to create the skeleton for our project and download it.
+The Spring Initializr will take care of creating the project structure for you, with the basic files and settings in place for the selected build tool.
+
+=== Other dependencies
+
+If you are looking at the project in Github, you might notice that there are some other dependencies in the `pom.xml`.
+A couple are for adding tests to the project, then one dependency for developer tools, and a couple more for test containers.
+
+More information on the testing functionality can be found in the https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#sdn.testing[documentation^].
+
+.Testing and dev tools dependencies
 [source,xml,subs="verbatim,attributes"]
 ----
-<dependencies>
-    <dependency>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-data-neo4j</artifactId>
-	</dependency>
-</dependencies>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-test</artifactId>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>io.projectreactor</groupId>
+    <artifactId>reactor-test</artifactId>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>junit-jupiter</artifactId>
+    <version>{testcontainers-version}</version>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>neo4j</artifactId>
+    <version>{testcontainers-version}</version>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-devtools</artifactId>
+    <scope>runtime</scope>
+    <optional>true</optional>
+</dependency>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-devtools</artifactId>
+    <scope>runtime</scope>
+    <optional>true</optional>
+</dependency>
```

Comment:
```suggestion
<dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-devtools</artifactId>
    <scope>runtime</scope>
    <optional>true</optional>
</dependency>
```
1. Duplication of the block.
2. Is this needed for the example? (Keeping it short)

Reply:
removed